lib/nerves_hub_link_common/message/update_payload.ex

```diff
@@ -0,0 +1,39 @@
+defmodule NervesHubLinkCommon.Message.UpdatePayload do
```

Comment:
```suggestion
defmodule NervesHubLinkCommon.Message.Update do
```

Does it work if we call this an `Update` message rather than an `UpdatePayload` message?

Reply:
I thought about this. I wanted to leave it `UpdatePayload` since the referenced NervesHubWeb file is called that as well.